### PR TITLE
[FIX] base_setup, web: fix image parsing error on save res_config

### PR DESCRIPTION
**Pull request by jvnm-dev**
### Issue

	- Have 2 languages installed
	- Install Invoicing & Website
	- Go in Settings
	- Search "logo" (image)
	- Upload a logo for your website
	- Save
	- Search "Default terms & conditions" (translatable field)
	- Click on the translate button (e.g. EN if you are in
	  english)

	A popup will appears saying that you should save your
	changes.

	- Click on Save & do not edit translation (not needed)
	- Search "logo"

	Image disappeared (because the value is it's size)
        and saving = parsing error

### Cause

	When saving, a read is called. By default, read has
	bin_size to true to avoid performances issues.
	It will return the image size instead of the content

	This is why the image disappeared.

### Solution

	As configuration is dirty, you are forced to save &
	it will always read with bin_size set to true.

	My solution would be to prioritize the context and
	set it to False for settings

**OPW-2208232**

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jvm-odoo** [comment]
Hi @aab-odoo , could you check this when you have a moment? Thanks!

**Comment by nim-odoo** [comment]
robodoo rebase-ff

File: addons/web/static/tests/views/form_tests.js [hunk 1]
```diff
@@ -415,8 +415,6 @@ QUnit.module('Views', {
                 // field, not sure why.  Maybe this test should be modified.
                 assert.deepEqual(args.args[1], ["foo", "display_name"],
                     "should only fetch requested fields");
-                assert.deepEqual(args.kwargs.context, {bin_size: true},
```
Comment (aab-odoo) [inline addons/web/static/tests/views/form_tests.js]: you could keep this I think... By default, bin_size is set to true in the context

File: addons/web/static/tests/views/form_tests.js [hunk 2]
```diff
@@ -5221,6 +5219,45 @@ QUnit.module('Views', {
         await assertFormContainsNButtonsWithSizeClass(6, 7);
     });
 
+    QUnit.test('bin_size in context', async function (assert){
```
Comment (aab-odoo) [inline addons/web/static/tests/views/form_tests.js]: can set bin_size to false in context

File: addons/web/static/tests/views/form_tests.js [hunk 3]
```diff
@@ -5221,6 +5219,45 @@ QUnit.module('Views', {
...
+            res_id: 1,
+            mockRPC: async function (route, args) {
+                assert.ok(args.kwargs.context.bin_size,
+                    "bin_size: true should always be in the context by default");
+                return this._super(route, args);
+            }
+        });
+        defaultForm.destroy();
+
+        var noBinSizeForm = await createView({
```
Comment (aab-odoo) [inline addons/web/static/tests/views/form_tests.js]: only keep this scenario

**Comment by jvm-odoo** [comment]
Should be good!

**Comment by aab-odoo** [comment]
@nim-odoo I agree with the JS changes

**Pull request closed without merging**